Code/Framework/AzCore/Tests/EBus/TestEventProcessingPolicy.cpp

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/EBus/IEventScheduler.h>
+#include <AzCore/EBus/EventSchedulerSystemComponent.h>
+#include <AzCore/EBus/ScheduledEvent.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Console/LoggerSystemComponent.h>
+#include <AzCore/Time/TimeSystem.h>
+#include <AzCore/Name/NameDictionary.h>
+#include <AzCore/UnitTest/TestTypes.h>
+
+namespace UnitTest
+{
+    class EventProcessingPolicyTests
+        : public AllocatorsFixture
+    {
+    public:
+        void SetUp() override
+        {
+            SetupAllocator();
+            AZ::NameDictionary::Create();
+        }
+
+        void TearDown() override
+        {
+            AZ::NameDictionary::Destroy();
+            TeardownAllocator();
+        }
+
+    };
```

Comment:
The IEVentScheduler, EVentSchedulerSystemComponent, ScheduleEVent, LoggerSystemComponent and NameDictionary aren't needed here.

Also a ScopedAllocatorSetupFixture should be used here to initialize the allocators

```suggestion
#include <AzCore/EBus/EBus.h>
#include <AzCore/Interface/Interface.h>
#include <AzCore/UnitTest/TestTypes.h>

namespace UnitTest
{
    class EventProcessingPolicyTests
        : public ScopedAllocatorSetupFixture
    {
    };
```